### PR TITLE
Updating service id

### PR DESCRIPTION
**Pull request by GeetikaBatra**

### CODE REVIEW

**Comment by GeetikaBatra** [comment]
@tisnik, please review and merge if necessary. Thanks in advance.

**Comment by jpopelka** [comment]
[test]

**Comment by GeetikaBatra** [comment]
Looks like component-feature is failing.

**Comment by invincibleJai** [comment]
@tisnik any help here 

`Failing scenarios:

  features/component_search.feature:19  Check the component search functionality for nonexistent component from PyPi ecosystem
`

**Comment by invincibleJai** [comment]
[test]

**Comment by tisnik** [comment]
@invincibleJai it was afaik caused by DNS issue on CI side

File: integration-tests/features/steps/three_scale.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def validate_result_post_registration(context):
     endpoints = check_and_get_attribute(json_data, "endpoints")
 
     prod_url = check_and_get_attribute(endpoints, "prod")
-    assert prod_url.startswith("http://")
+    assert prod_url.startswith("https://")
 
-    stage_url = check_and_get_attribute(endpoints, "stage")
-    assert stage_url.startswith("http://")
+    # stage_url = check_and_get_attribute(endpoints, "stage")
+    # assert stage_url.startswith("http://")
```
Comment (tisnik) [inline integration-tests/features/steps/three_scale.py]: don't we need to check for "https?:// for stage?

**Comment by tisnik** [comment]
LGTM for me. Thanks!

File: integration-tests/features/steps/three_scale.py (hunk 1 above)
Comment (invincibleJai) [reply integration-tests/features/steps/three_scale.py]: yes it has TLS

**Comment by miteshvp** [comment]
LGTM

**Pull request merged**